<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CCU Salary     by toothpick</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>531</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="account">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>10</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>40</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="identity">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>70</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>輸入帳號</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>輸入密碼</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>選擇登入身分</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_4">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>計畫編號</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>190</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>年</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="year">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>190</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>190</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>月</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>320</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>320</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>320</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>320</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>448</x>
      <y>320</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>350</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>350</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_8">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>350</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_9">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>350</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_10">
    <property name="geometry">
     <rect>
      <x>448</x>
      <y>350</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_11">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>380</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_12">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>380</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_13">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>380</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>13</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_14">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>380</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>14</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_15">
    <property name="geometry">
     <rect>
      <x>448</x>
      <y>380</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_16">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>410</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>16</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_17">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>410</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>17</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_18">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>410</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>18</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_19">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>410</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>19</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_20">
    <property name="geometry">
     <rect>
      <x>448</x>
      <y>410</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_21">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>440</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>21</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_22">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>440</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>22</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_23">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>440</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>23</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_24">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>440</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>24</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_25">
    <property name="geometry">
     <rect>
      <x>448</x>
      <y>440</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>25</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_26">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>470</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>26</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_27">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>470</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>27</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_28">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>470</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>28</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_29">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>470</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>29</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_30">
    <property name="geometry">
     <rect>
      <x>448</x>
      <y>470</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_31">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>500</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>31</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>每日工作時數</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="workHours">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>190</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>230</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>工作內容：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="workIn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="planNumComboBox">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>160</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="month">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>190</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>01</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>05</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>06</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>09</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>100</y>
      <width>271</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>登入</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="createLogs">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>530</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>產生工作日誌</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="createPrintButton">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>530</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>產生日誌批號</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="logOutButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>530</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>登出</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>280</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>其他 (非必填)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1116</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>account</tabstop>
  <tabstop>password</tabstop>
  <tabstop>identity</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>planNumComboBox</tabstop>
  <tabstop>year</tabstop>
  <tabstop>month</tabstop>
  <tabstop>workHours</tabstop>
  <tabstop>workIn</tabstop>
  <tabstop>createLogs</tabstop>
  <tabstop>createPrintButton</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>logOutButton</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>checkBox_4</tabstop>
  <tabstop>checkBox_5</tabstop>
  <tabstop>checkBox_6</tabstop>
  <tabstop>checkBox_7</tabstop>
  <tabstop>checkBox_8</tabstop>
  <tabstop>checkBox_16</tabstop>
  <tabstop>checkBox_17</tabstop>
  <tabstop>checkBox_18</tabstop>
  <tabstop>checkBox_19</tabstop>
  <tabstop>checkBox_20</tabstop>
  <tabstop>checkBox_21</tabstop>
  <tabstop>checkBox_22</tabstop>
  <tabstop>checkBox_23</tabstop>
  <tabstop>checkBox_24</tabstop>
  <tabstop>checkBox_25</tabstop>
  <tabstop>checkBox_26</tabstop>
  <tabstop>checkBox_27</tabstop>
  <tabstop>checkBox_28</tabstop>
  <tabstop>checkBox_29</tabstop>
  <tabstop>checkBox_30</tabstop>
  <tabstop>checkBox_31</tabstop>
  <tabstop>checkBox_11</tabstop>
  <tabstop>checkBox_12</tabstop>
  <tabstop>checkBox_13</tabstop>
  <tabstop>checkBox_14</tabstop>
  <tabstop>checkBox_10</tabstop>
  <tabstop>checkBox_15</tabstop>
  <tabstop>checkBox_9</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
